knx/knxnet/proto.go

```diff
@@ -96,10 +98,10 @@ func AllocAndPack(srv ServicePackable) []byte {
 	return buffer
 }
 
-// These are errors that might occur during unpacking.
+// These are errors that might occur during unpacking of the header.
 var (
-	ErrHeaderLength  = errors.New("header length is not 6")
-	ErrHeaderVersion = errors.New("protocol version is not 16")
+	ErrHeaderLength  = errors.New("header length is not 0x6")
```

Comment:
This is not very useful. I don't think lengths should be rendered in hexadecimal.

Reply:
ditto above.